app.json

```diff
@@ -8,5 +8,6 @@
       "description": "Find this at https://dashboard.stripe.com/account/apikeys (it'll look like sk_test_****)",
       "required": true
     }
-  }
+  },
+  "stack": "heroku-18"
```

Comment:
Will there be any issue upgrading the ruby version to the latest instead of pining to Heroku stack?

Reply:
I think we can look at upgrading the Ruby version as a separate effort.